src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -250,7 +260,8 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
       stakingAssetAccountId,
       stakingAssetId,
       stakingAmountCryptoBaseUnit: toBaseUnit(amountCryptoPrecision, stakingAsset.precision),
-      runeAddress,
+      // typescript is borked, one of them is defined because of the early return
+      runeAddress: currentRuneAddress ? currentRuneAddress : runeAddress ?? '',
```

Comment:
if `currentRuneAddress` is equal to ""

`currentRuneAddress ?? runeAddress` returns `currentRuneAddress` which is obviously wrong... JavaScript please :(

Reply:
@NeOMakinG nullish coalescing no works with falsy values (e.g empty string), only `false | undefined` - if you short circuit instead, this will be happy

Reply:
Changed in [0140283](https://github.com/shapeshift/web/pull/7734/commits/0140283f2ea4458462c0a38261b3b119b9a2eb8c)